src/molecules/BarSm.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Composition } from 'atomic-layout'
+import { IoIosSearch, IoMdPerson, IoMdCart } from 'react-icons/io'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  background-color: #262626;
+  justify-content: space-around;
```

Comment:
`justify-content` does nothing if its element has no `display:flex` or `display:grid`. Please remove, or add the missing display property.